src/ragas/integrations/langgraph.py

```diff
@@ -0,0 +1,71 @@
+import json
+from typing import List, Union
+
+from langchain_core.messages import AIMessage, HumanMessage, SystemMessage, ToolMessage
+
+import ragas.messages as r
+
+
+def convert_to_ragas_messages(
```

Comment:
Shouldn't we have a test for this function? @jjmachan @sahusiddharth

Reply:
yep - that would be great

Reply:
On it, making test cases.

I’m unsure whether I should place them in the unit tests (since it’s a single function) or in e2e tests (since this function currently represents the entire model).